docs/source/developers/python.rst

```diff
@@ -148,7 +148,7 @@ Pull in the test data and setup the environment variables:
    git submodule update
    export PARQUET_TEST_DATA="${PWD}/cpp/submodules/parquet-testing/data"
    export ARROW_TEST_DATA="${PWD}/testing/data"
-
+   popd
```

Comment:
This is a nitpick: without it you need to do `popd` by hand before the next step